Antonio - MarineAntonio/week10/econ490_week10_Marine.do

```diff
@@ -151,17 +152,131 @@ use "Tz_elec_15_clean", clear
 rename *_15 *  // like before, dropping all of the suffixes (_15)
 drop total_candidates ward_total_votes // drop same vars again
 
-* OBSERVATION: the 'ward' variable alone doesn't uniquely identify observations, so there must be multiple wards with the same name. Stata doesn't give me an error if I merge using district and ward OR region, district, and ward. I'm not positive which to do!
-* merge 1:1 region district ward using `tanz10' // when I merge using all three, I notice that there are many wards with the same name in the same region OR district, where one is parentless and the other is childless. This seems suspicious.
-merge 1:1 district ward using `tanz10' // ??
+* OBSERVATION: the 'ward' variable alone doesn't uniquely identify observations, so there must be multiple wards with the same name.
+merge 1:1 region district ward using `tanz10' // After merging, it can be seen that there are many wards with the same name. The suspicious case is when these wards are in either the same region OR district, where one is parentless and the other is childless. This happens a lot.
+* sort ward _merge

```

Comment:
Hi Antonio! Apologies for my late review. Please what to you mean by this being a suspicious case?